### DISCUSSION TITLE
Adding chain logos to chain definitions

### DISCUSSION THREAD

**Discussion by haardikk21** [wevm/viem#2391] [Idea / Feature Request] [answered]
Viem has a large repository of chain definitions which are often used (at least by me) in the backend when doing chain-agnostic data parsing/data collection

Sometimes when doing that then it's helpful to spin up a frontend to make sense of the data. For visual appeal, or if you plan on sharing that frontend with others, a task that can be quite annoying is setting up a mapping of all chain id's to logos (which I need to gather from a variety of sources)

With viem already having a great place to put it, I'd love to see an `imageUrl` or `logo` property added to chain definitions to save us that headache. Whether the images themselves should be hosted on the `wevm` GitHub org, or reference image links from across the internet (CryptoLogos, official brand kits, X, Farcaster, etc) - I don't have an opinion on 

Sample use case: looping over all chains supported by Viem and figuring out native currency balances on those chains across a few different wallets to try and see if i have leftover dust somewhere i've forgotten about

**Comment by tmm** [comment, marked as answer]
Unfortunately we are not interested in Viem being the source of truth for this info at the moment. If something changes in the future, I will update this thread.
